src/gwt/src/org/rstudio/studio/client/common/vcs/ShowPublicKeyDialog.java

```diff
@@ -58,8 +59,8 @@ protected Widget createMainWidget()
       int mod = BrowseCap.hasMetaKey() ? KeyboardShortcut.META : 
                                          KeyboardShortcut.CTRL;
       String cmdText = new KeyCombination("c", 'C', mod).toString(true);
-      HTML label = new HTML("Press " + cmdText + 
-                            " to copy the key to the clipboard");
+      HTML label = new HTML(constants_.pressLabel() + cmdText +
+                            " " + constants_.copyKeyToClipboardLabel());
```

Comment:
String substitution, not concatenation.

"Press {0} to copy the key to the clipboard"